### ISSUE TITLE
Make config reffered to absolute path

### ISSUE THREAD

**Issue by HarishJella** [Kubessandra/react-google-calendar-api#37]
./node_modules/react-google-calendar-api/ApiCalendar.js
Cannot find module: '../../../apiGoogleconfig.json'. Make sure this package is installed.

make it absolute path

**Comment by vlad-dewitt** [comment]
../../../apiGoogleconnfig.json. Really? where is it? It's outside the project... what??

**Comment by vlad-dewitt** [comment]
@Kubessandra ???

**Comment by Kubessandra** [comment]
@vlad-dewitt I never had the time to implement proper path resolving.

From the README:
Create a file apiGoogleconfig.json in the root directory with your googleApi clientId and ApiKey.
https://console.developers.google.com/flows/enableapi?apiid=calendar.

```json
{
  "clientId": "<CLIENT_ID>",
  "apiKey": "<API_KEY>",
  "scope": "https://www.googleapis.com/auth/calendar",
  "discoveryDocs": ["https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest"]
}
```

**Comment by vlad-dewitt** [comment]
@Kubessandra, but it says "Cannot find module: '../../../apiGoogleconfig.json'", because this path is outside of project dir

**Comment by Kubessandra** [comment]
@vlad-dewitt yes unfortunately, this is outside of the project dir.
I'm creating a PR right now to make it at the root dir, I don't know why I made it this way the first time.

**Comment by vlad-dewitt** [comment]
@Kubessandra, got it. Many thanks!

**Comment by Kubessandra** [comment]
The problem was in the import `react-google-calendar/ApiCalendar.ts` because the file was using `../../../` and this import was good only in the build directory(`node_modules/react-google-calendar-api/build/ApiCalendar.js`.

This is now fixed, in the version 1.2.0.
But this is always better to use the default import using `import ApiCalendar from 'react-google-calendar-api'`.

@vlad-dewitt Keep me informed if you still have the issue or not, thank you.

If you still want to use the Typescript import, this is now: `import ApiCalendar from 'react-google-calendar-api/src/ApiCalendar`.

**Comment by craigjmidwinter** [comment]
Sorry to bump a closed issue, but I'm using this with gatsby and it seems like no matter where I place my config, it can't resolve it during the build. 
```
error undefined failed

Can't resolve '../../../apiGoogleconfig.json' in '/home/craig/workspace/display-v2/node_modules/react-google-calendar-api/build'

If you're trying to use a package make sure that '../../../apiGoogleconfig.json' is installed. If you're trying to use a local file make sure that the path is correct.
failed Building development bundle - 2.461s
verbose 4.379988061 Transition to "waiting"
verbose 4.403193194 Webpack file changed: /home/craig/workspace/display-v2/.cache/api-runner-browser-plugins.js
verbose 4.403837198 Transition to "waiting" > "aggregatingFileChanges"
verbose 4.610740742 Transition to "runningQueries"
success onPreExtractQueries - 0.030s
success extract queries from components - 0.010s
verbose 4.650834875 Transition to "runningQueries" > "waitingPendingQueries"
verbose 4.702405219 Transition to "runningQueries" > "writingRequires"
success write out requires - 0.001s
verbose 4.703253636 Transition to "runningQueries" > "calculatingDirtyQueries"
verbose 4.703605761 Transition to "runningQueries" > "runningStaticQueries"
verbose 4.703912993 Transition to "runningQueries" > "runningPageQueries"
verbose 4.704357654 Transition to "runningQueries" > "waitingForJobs"
verbose 4.704719329 Transition to "runningQueries" > "done"
verbose 4.707299947 Transition to "recompiling"
error undefined failed

Can't resolve '../../../apiGoogleconfig.json' in '/home/craig/workspace/display-v2/node_modules/react-google-calendar-api/build'

If you're trying to use a package make sure that '../../../apiGoogleconfig.json' is installed. If you're trying to use a local file make sure that the path is correct.
failed Re-building development bundle - 0.094s
verbose 4.804546869 Transition to "waiting"
^C
display-v2 on  calendar [!?] is 📦 v0.1.0 via ⬢ v12.17.0 took 1m47s 
❯ pwd
/home/craig/workspace/display-v2

display-v2 on  calendar [!?] is 📦 v0.1.0 via ⬢ v12.17.0 
❯ ls
appGoogleconfig.json  gatsby-config.js  jsconfig.json  node_modules  package-lock.json  README.md  static          yarn.lock
appgoogleconfig.json  gatsby-node.js    LICENSE        package.json  public             src        yarn-error.log

```
